src/PluginLoader.php

```diff
@@ -59,7 +59,14 @@ public static function getGeneratedPath(): string
      */
     public function getInstances(): array
     {
-        return array_diff_key($this->plugins, array_flip($this->disabled));
+        $plugins = array_filter(
+            $this->getInstances(),
```

Comment:
How can this recursion work?

Reply:
```suggestion
            $this->plugins,
```

Reply:
fixed in 3f559bc235ea2288d1b5ae3f00bbe52781427fdf